pages/products/[id].jsx

```diff
@@ -18,68 +15,7 @@ function ShowProduct(props) {
     createdAt: Date(),
   };
 
-  return (
-    <div className="upload-show">
-      <SimpleGrid columns={3}>
-        <Product product={product} displayButton={false} />
-
-        <Text
-          center
-          py={5}
-          width={700}
-          style={{
-            display: "block",
-            verticalAlign: "middle",
-            marginTop: "25%",
-          }}
-        >
-          <b>Created by:</b> {product.creator}
-          <br />
-          <b>Current owner:</b> {product.owner}
-          <br />
-          <b>Description:</b> {product.description}
-          <br />
-          <b>Categories:</b> {product.categories}
-          <br />
-          <b>Creation date:</b> {product.createdAt}
-          <br />
-          <b>Most recent update date:</b> {product.updatedAt}
-          <br />
-          <br />
-          <div>
-            <Button
-              style={{ marginRight: "20px" }}
-              leftIcon={<EditIcon />}
-              colorScheme="purple"
-              onClick={() => console.log("Edit pressed")}
-            >
-              Edit
-            </Button>
-            <Button
-              style={{ marginRight: "20px" }}
-              leftIcon={<DeleteIcon />}
-              colorScheme="purple"
-              variant="outline"
-              onClick={() => console.log("Delete pressed")}
-            >
-              Delete
-            </Button>
-
-            <Link href={"/"} passHref>
-              <Button
-                leftIcon={<ArrowBackIcon />}
-                colorScheme="purple"
-                variant="outline"
-                onClick={() => console.log("Back pressed")}
-              >
-                Back
-              </Button>
-            </Link>
-          </div>
-        </Text>
-      </SimpleGrid>
-    </div>
-  );
+  return <EditableProduct product={product} />;

```

Comment:
Not sold on having `EditableProduct` as a separate function component, you could bring that logic over here, and I'm starting to notice some [prop drilling](https://kentcdodds.com/blog/prop-drilling). But you could leave it like this, no big deal 👍🏼